<launch>
<arg name="robot_name" default="asl_gremlin1"/>

<node   ns="$(arg robot_name)"
        pkg="controller"
        name="backstepping_controller"
        type="backstepping_controller_node"
        respawn="true"
        output="screen" />

<node   ns="$(arg robot_name)"
        pkg="controller"
        name="angular_vel_to_pwm"
        type="angular_vel_to_pwm_node"
        respawn="true"
        output="screen" />
        
 <node   ns="$(arg robot_name)"
        pkg="controller"
        name="cmd_pwm_to_arduino_pwm"
        type="cmd_pwm_to_arduino_pwm"
        respawn="true"
        output="screen" />      

</launch>
